commands/motor/MotorIdle.java

```diff
@@ -18,37 +21,36 @@
 	 * @param motor
 	 */
 	public MotorIdle(Motor motor) {
-		super("MotorIdle");
 		this.motor = motor;
-		requires(motor);
-		setInterruptible(true); // default command
 		LogKitten.d("MotorIdle created");
 	}
 
 	@Override
-	protected void initialize() {
+	public void initialize() {
 		motor.set(0);
 		LogKitten.d("MotorIdle initialized");
 	}
 
 	@Override
-	protected void execute() {
+	public void execute() {
 		motor.set(0);
 		LogKitten.d("MotorIdle executing");
 	}
 
 	@Override
-	protected boolean isFinished() {
+	public boolean isFinished() {
 		return false; // default command
 	}
 
 	@Override
-	protected void end() {
-		LogKitten.d("MotorIdle ended");
+	public void end(boolean interrupted) {
+		LogKitten.d("MotorIdle interrupted");
 	}
-
 	@Override
-	protected void interrupted() {
-		LogKitten.d("MotorIdle interupted");
+	public Set<Subsystem> getRequirements() {
+		Set<Subsystem> motors = new HashSet<Subsystem>();
+		motors.add(motor);
+		// TODO Auto-generated method stub
```

Comment:
Generally good practice to removd the TODO Auto-generated method stub when you have added the desired functionality to the method.